functions/twist_angle.h

```diff
@@ -24,4 +24,25 @@ class TwistAngleSVF {
 template<int N = 2, int OFFSET = 0>
 using TwistAngle = SingleValueAdapter<TwistAngleSVF<N, OFFSET>>;
 
+// Usage: TwistAcceleration<MAX>
+// Returns 0-32768 based on acceleration of twist in one direction
+// MAX : Maximum acceleration needed to return 32768
+// returned value: FUNCTION, same for all leds
+
+template<class MAX>
+class TwistAccelerationSVF {
+ public:
+  void run(BladeBase* blade) { max_.run(blade); }
+  int calculate(BladeBase* blade) {
+    float accel = fusor.twist_accel();
+    if (accel < 0) accel *= -1;
```

Comment:
This is the same as fabs() and not what I meant.
I meant:
  if (fusor.gyro().z < 0) accel = -accel;

(if that doesn't work, try > instead of <)

Reply:
Fixed